book/website/ethical-research/ethics-case-study.md

```diff
@@ -0,0 +1,134 @@
+For example, the data employed may be filled with ["missing data sets"](https://github.com/MimiOnuoha/missing-datasets), and the methodology can be problematic if built on unchecked assumptions.
+
+**Responding to this question** requires describing the project's data and methodology in a detailed manner.
+Justification must also be provided as to *why* the outlined data are necessary and the described methodology is the best for the job.
+
+A more robust response to this question will describe how the data will be managed for the purpose of the methodology described.
+To this effect, it is advisable to share a data management plan.
+It is also helpful to describe how the data and methodology will be made available to other researchers.
+
+Therefore, responding to this question requires providing:
+
+1. Detail about the data and methodologies you will employ;
+2. Justification for the choice of data and methodologies;
+3. Reference to a data management plan -- consider the [Research Data Management chapter](https://the-turing-way.netlify.app/reproducible-research/rdm.html);
+4. Explanation as to how your chosen tools and methods allow for reproducible research -- consider the benefits listed in the [Added Advantages](https://the-turing-way.netlify.app/reproducible-research/overview/overview-benefit.html?highlight=method) subchapter.
+
+### Consent
+>How will you ensure any personal data collected for your research is done so with freely-given informed consent?
+
+**Justification.** The purpose of ensuring consent for the collection, use or reuse of personal data is to respect the autonomy of individuals and operate in line with pertinent legislation.
+It is also an important concept enshrined in law.
+In the General Data Protection Regulation (GDPR), for example, *consent* is defined as:
+
+>“any freely given, specific, informed and unambiguous indication of the data subject’s wishes by which he or she, by a statement or by a clear affirmative action, signifies agreement to the processing of personal data relating to him or her" ([ICO](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/consent/what-is-valid-consent/))
+
+**Responding to this question** entails reflection on how data are sourced -- where they come from and who they pertain to.
+Setting the legal discussions aside, this question requires following one of two approaches:
+1. If employing data that is readily available (perhaps online or provided by a third party), it is important we understand the terms and conditions whereby those data are made available. The point here is to confirm that the data will be used following the purpose for which they were collected. In this respect, a useful subchapter in *The Turing Way* is that of *[Data Licenses](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html?highlight=licen)*, which outlines different ways online data might be held.
```

Comment:
Previous internal (or external) links were not italicised, but "Data Licences" in this section was. I don't have a preference which way to go, just a request for consistency 😄

Reply:
Sorted! Thanks!